source/1_engine/rex_engine/include/rex_engine/diagnostics/stacktrace.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#ifdef REX_PLATFORM_WINDOWS
+  #include "rex_engine/diagnostics/win/win_stacktrace.h"
```

Comment:
Do we want these conditional compilation statements or do we exclude the file in general from the build?

Reply:
Shouldn't there be a platform header somewhere?

Reply:
I created an issue for excluding platform specific code from build.
I don't know what you mean with platform header